bundled/tool/server.py

```diff
@@ -11,6 +11,11 @@
 import traceback
 from typing import Any, Callable, Dict, List, Optional, Sequence, Union
 
+from pygls import lsp, protocol, server, uris, workspace
+
+import jsonrpc
+import utils
+
```

Comment:
These cannot go here, they will fail to load. They have to go back to their original location.

Reply:
Removed the imports (https://github.com/microsoft/vscode-pylint/pull/189/commits/183834bfa328a903c0ef28c72fdb35230df86250)